{{cookiecutter.project_slug}}/Makefile

```diff
@@ -46,7 +46,7 @@ docker-compose.yml: $(METADATA) ## generates docker-compose
 
 define _docker_compose_build
 export DOCKER_BUILD_TARGET=$(if $(findstring -devel,$@),development,$(if $(findstring -cache,$@),cache,production)); \
-	docker buildx bake  --file docker-compose.yml $(if $(findstring -nc,$@),--no-cache,);
+	docker compose -f docker-compose.yml build $(if $(findstring -nc,$@),--no-cache,);

```

Comment:
wondering: I wonder why they have now compose and bake. not sure what docker is planning with this.